Kubernetes/legos/k8s_get_service_with_no_associated_endpoints/k8s_get_service_with_no_associated_endpoints.py

```diff
@@ -0,0 +1,62 @@
+#
+# Copyright (c) 2023 unSkript.com
+# All rights reserved.
+#
+import pprint 
+
+from typing import Tuple
+from kubernetes import client
+from kubernetes.client.rest import ApiException
+from pydantic import BaseModel, Field
+
+
+class InputSchema(BaseModel):
+    namespace: str = Field(
+        default = "",
+        title = "K8S Namespace",
+        description = "Kubernetes Namespace Where the Service exists"
+    )
+
+def k8s_get_service_with_no_associated_endpoints_printer(output):
+    if output is None:
+        return 
+    
+    print(output)
+
+def k8s_get_service_with_no_associated_endpoints(handle, namespace: str = "") -> Tuple:
+    """k8s_get_service_with_no_associated_endpoints This function returns Services that
+       do not have any associated endpoints.
+
+       :type handle: Object
+       :param handle: Object returned from the task.validate(...) function
+
+       :type namespace: str
+       :param namespace: String, K8S Namespace as python string 
+
+       :rtype: Tuple Result in tuple format.
+    """
+    if handle.client_side_validation != True:
+        raise Exception(f"K8S Connector is invalid {handle}")
+    
+    v1 = client.CoreV1Api(api_client=handle)
+    
+    services = v1.list_service_for_all_namespaces().items
```

Comment:
These list commands, we dont seem to handle pagination? Check other places too.
Use https://github.com/kubernetes-client/python/blob/master/kubernetes/docs/CoreV1Api.md#list_namespaced_service for namespaced services, instead of doing it in memory.

Reply:
Will be handled as a seperate issue.

Reply:
Please ensure that you take care of using the namespaced api as well there as i mentioned above